forge/test/models/pytorch/vision/regnet/test_regnet.py

```diff
@@ -0,0 +1,54 @@
+# SPDX-FileCopyrightText: © 2024 Tenstorrent AI ULC
+
+# SPDX-License-Identifier: Apache-2.0
+import pytest
+from transformers import AutoImageProcessor, RegNetModel, RegNetForImageClassification
+import forge
+from forge.verify.verify import verify
+from .utils.image_utils import preprocess_input_data
+
+
+@pytest.mark.nightly
+@pytest.mark.model_analysis
+@pytest.mark.xfail(
+    reason="RuntimeError: TT_FATAL @ tt-forge-fe/third_party/tt-mlir/third_party/tt-metal/src/tt-metal/ttnn/cpp/ttnn/tensor/tensor_utils.cpp:474: new_volume == old_volume"
+)
+@pytest.mark.parametrize("variant", ["facebook/regnet-y-040"])
+def test_regnet(variant):
+
+    # Load RegNet model
+    framework_model = RegNetModel.from_pretrained("facebook/regnet-y-040")
+
+    # Preprocess the image
+    image_url = "http://images.cocodataset.org/val2017/000000039769.jpg"
+    inputs = preprocess_input_data(image_url, variant)
+
+    # Compiler test
+    compiled_model = forge.compile(
+        framework_model, sample_inputs=[inputs[0]], module_name="pt_" + str(variant.split("/")[-1].replace("-", "_"))
```

Comment:
Can you change `sample_inputs=[inputs[0]]` to `sample_inputs=inputs` as `preprocess_input_data` returns list of one tensor, so there is no need to take first element and wrap it up into list again :))